src/app/contexts/UserContext/index.test.tsx

```diff
@@ -61,4 +64,77 @@ describe('UserContext', () => {
       {},
     );
   });
+  describe('ckns_mvt cookie', () => {
+    const cookieSetterSpy = jest.spyOn(Cookie, 'set');
+    const cookieGetterSpy = jest.spyOn(Cookie, 'get');
+    const isOperaProxySpy = jest.spyOn(isOperaProxy, 'default');
+    const onClientSpy = jest.spyOn(onClient, 'default');
+
+    beforeEach(() => {
+      jest.clearAllMocks();
```

Comment:
Should this be reset instead of clear?
```suggestion
      jest.resetAllMocks();
```

Reply:
I thought clearAllMocks, clears all usage data of the mock and resetAllMocks also resets the implementation, in this context is clear not what we want?

Reply:
I'm not sure either, I just know in the past I've had issues whereby a single test will pass when run on its own, and then will fail when run with all of the other tests. In my experience, it was because of the incorrectly using clear or reset (can't recall which one!). But since the NextJS tests are now working with your fix for the document.domain, maybe we don't need to worry about this.